app/views/tweets/tweets.erb

```diff
@@ -0,0 +1,29 @@
+<% if Helper.logged_in?(session) %>
+  <h1>Welcome, <%= @user.username.capitalize %></h1>
+
+  <h3>Feed</h3>
+
+  <% User.all.map do |u| %>
```

Comment:
In a MVC pattern we try to avoid database calls in the View. I would set this to `@users = User.all` in the `get '/tweets' do` method